webapp/src/Controller/SecurityController.php

```diff
@@ -103,58 +105,62 @@ public function registerAction(
         $registration_form->handleRequest($request);
         if ($registration_form->isSubmitted() && $registration_form->isValid()) {
             $plainPass = $registration_form->get('plainPassword')->getData();
-            $password  = $passwordHasher->hashPassword($user, $plainPass);
-            $user->setPassword($password);
-            if ($user->getName() === null) {
-                $user->setName($user->getUsername());
-            }
+            if (strlen($plainPass) < UserController::MIN_PASSWORD_LENGTH) {
```

Comment:
Wouldn't it make more sense to store this constant here in the security controller, or possibly in some other global place?

Reply:
Do you have a good suggestion? Since we also use it in the jury controller.

Reply:
Maybe in `webapp/config/static.yaml.in`? Not sure whether we should also make it configurable from `configure` or so...

Reply:
I do like that idea, but then maybe we do that in a separate issue / PR?

Reply:
I moved it to the same place as the enabling of removed intervals. Made sense to me.